### PR TITLE
switch feed to flatlist

### PR DESCRIPTION
**Pull request by JakeSteinebronn**
# Changes
 - Add keys to a bunch of stuff in the profile top component to make it not have a key error
# Issue ticket number and link

### CODE REVIEW

**Comment by JakeSteinebronn** [comment]
Draft, because I also have to add keys to things in the route view. Why didn't we have to do this before? Is there a way around it without just ignoring the error?

File: components/media/Feed.tsx [hunk 1]
```diff
@@ -68,36 +46,56 @@ const Feed = ({
...
-        }
+    <FlatList
+      ListHeaderComponent={topComponent}
+      data={posts}
+      onEndReached={loadNextPosts}
+      onEndReachedThreshold={0.1}
+      ListFooterComponent={renderSpinner}
+      ItemSeparatorComponent={Divider}
+      bgColor={baseBgColor}
+      keyExtractor={(item, index) => {
```
Comment (abrugal) [inline components/media/Feed.tsx:63]: Weird, I didnt get a key component error. Also you should use the id of the post as the key instead of the index

Comment (JakeSteinebronn) [reply components/media/Feed.tsx]: If you can find a way to make this change without a key component error, i'll happily abandon this diff lol. Maybe a skissue? but I couldn't get it to not throw that error without adding keys everywhere

**Review by tylerhm — APPROVED** [review]
Nice nice! Jacob and I found the key bug offline, and it is now addressed.

For people reading: The top component must be a function rather than a *pre-built* component.

**Pull request merged**